packages/citrix-adc-istio-ingress-gateway/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,9 +1,25 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -13,3 +13,6 @@
- sources:
- - https://github.com/citrix/citrix-istio-adaptor
- version: 1.2.1
+@@ -1,15 +1,21 @@
 +annotations:
 +  catalog.cattle.io/certified: partner
++  catalog.cattle.io/display-name: citrix-adc-istio-ingress-gateway
```

Comment:
The display-name is the name displayed for the chart in the UI, so it should be capitalized and space separated rather than hyphenated. For example, `Citrix ADC Istio Ingress Gateway`

CC @alex-isv